src/utils/sanitizeContent.js

```diff
@@ -0,0 +1,18 @@
+import DOMpurify from "isomorphic-dompurify";
+
+export const sanitizeContent = (data, props) => {
+  const sanitizedContent = DOMpurify.sanitize(data.content);
+  
+  let chunk = { ...data, content: sanitizedContent };
+  let tokens = chunk.content.match(/\{{(.*?)\}}/g);
+
+  let parsedChunk = chunk.content;
+
+  if (tokens !== null) {
+    tokens.forEach(function(token) {
```

Comment:
How bout `tokens?.forEach` instead of 

```javascript
if (tokens != null) {
  tokens.forEach(function(token) {
}
```

Reply:
@jorenrui not sure if this is supported now on ES6, though. I think coalescing is only available on ES2020 as well as TypeScript.

Reply:
Yeah. I'll skip this for now.